plasmapy/transport/classical/epperlein_haines.py

```diff
@@ -0,0 +1,238 @@
+c0p=np.array([1.93, 1.89, 1.66, 1.31, 1.12, 1.04, 1.02, 0.875, 0.77, 0.722, 0.674, 0.605, 0.566, 0.502, 0.457])
+c1p=np.array([2.31, 3.78, 4.76, 4.63, 4.62, 4.83, 5.19, 4.74, 4.63, 4.7, 4.64, 4.65, 4.81, 4.71, 4.81])
+c2p= np.array([5.3, 7.78, 8.88, 8.8, 8.8, 8.96, 9.24, 8.84, 8.71, 8.73, 8.65, 8.6, 8.66, 8.52, 8.53])
+γ0x = np.array([6.071, 15.75, 25.65, 34.95, 43.45, 51.12, 58.05, 64.29, 75.04, 83.93, 91.38, 107.8, 124.3, 145.2, 172.7])
+γ0pp = np.array([4.01, 2.46, 1.13, 0.628, 0.418, 0.319, 0.268, 0.238, 0.225, 0.212, 0.202, 0.200, 0.194, 0.189, 0.186])
+γ1pp = np.array([2.5] * 15)
+c0pp = np.array([.661, .156, .0442, .018, .00963, .00625, .00461, .00371, .003, .00252, .00221, .00185, .00156, .0013, .00108])
+c1pp = np.array([.931, .398, .175, .101, .0702, .0551, .0465, .041, .0354, .0317, .0291, .0256, .0228, .0202, .018])
+c2pp = np.array([2.5, 1.71, 1.05, .775, .646, .578, .539, .515, .497, .482, .471, .461, .45, .44, .43])
+
+
+def _find_nearest(Z):
+    """
+    Finds the nearest Z-value in the EH tables for a given Z. Prints a 
+    warning if the Z found is not equal to the Z requested.
+
+    Parameters
+    ----------
+    Z : float
+        An integer charge
+
+    Returns
+    -------
+    i : int
+        The index of the closest Z in the EH tables
+
+    """
+    i = np.argmin(np.abs(Ztable - Z))
+    if Ztable[i] != Z:
+        warnings.warn(f"Value Z = {Z} is not in the Epperlein-Haines table. "
+                      f"Using the nearest value, Z = {Ztable[i]}. "
+                      f"The values in the table are {Ztable}.",
+                      RuntimeWarning)
+    return i
+
+
+class EpperleinHainesCoefficents(AbstractClassicalTransportCoefficients):
+    
+    def norm_alpha_para(self):
+        """
+        Calculates the normalized alpha_para coefficient in terms of the
+        dimensionless Hall parameter and the ionization fraction.
+    
+        Parameters
+        ----------
+        chi : float 
+            The dimensionless hall parameter (ratio of the electron gyrofrequency
+            and the electron-ion collision frequency).
+            
+        Z : float
+            Ionization fraction. The value will be coerced to the nearest value
+            from the Epperlein-Haines tables: 
+            Z = [1, 2, 3, 4, 5, 6, 7, 8, 10, 12, 14, 20, 30, 60, ∞]
+    
+        Returns
+        -------
+        norm_alpha_para, u.Quantity
+            The resistivity parallel to (or in the absence of) a magnetic field
+            
+        Notes
+        -----
+        For details, see "Plasma transport coefficients in a magnetic field by 
+        direct numerical solution of the Fokker–Planck equation" by
+        E. M. Epperlein and M. G. Haines, DOI: `10.1063/1.865901`
+    
+        .. _`10.1063/1.865901`: https://aip.scitation.org/doi/10.1063/1.865901
+    
+        """
+        i = _find_nearest(self.Z)
+        return α0[i]*np.ones(self.chi.size)
```

Comment:
So this is essentially nearest neighbor interpolation, right? Might cause nonphysical discrete jumps as you sweep the parameter space. Wouldn't we want something like scipy splines instead? Polynomial fits?

Reply:
Yeah I wanted to raise that question. I do want there to be some way to extract exactly the data from the paper (for historical purposes?). But some sort of fits in Z would be good. 

For models where we have the actual data (transport coefficients tabulated as functions of chi and Z rather than polynomial fits) I have some code that just interpolates the values from that using scipy's `interp2d`. I suppose we could generate these tables from the polynomial fits and then apply the same method here?

Reply:
As long as that doesn't introduce any heavy error terms... sure, should be good enough!

Reply:
Do you mean relative to the original data? Sure, we can check that.

Reply:
Another option would be to implement both the explicit polynomials (exactly as given in the paper) and the fit, eg. `EpperleinHainesCoefficients` and `EpperleinHainesCoefficientsPolynomialFit`. Then we could write tests to ensure that the interpolated values in the former were within some margin of the fits given in the papers?